makefile

```diff
@@ -65,19 +65,28 @@ start-backend-prod:
 	python -m xonsh make.xsh start-backend-prod
 
 
+# Starts up a local postgres container from the docker-compose.yml
+run-local-db:
+	python -m xonsh make.xsh run-local-db
+
+
 # Use the "database-backup" service in the "docker-compose.yml" file to backup
 # all of the tables in the instace of Postgres running locally.
-backup-database-dev:
-	python -m xonsh make.xsh backup-database-dev
+backup-local-db:
```

Comment:
Nice! Did you run `make make` to generate these new targets in this makefile?

Reply:
can't say I did... I'll look into that for future reference